src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -297,6 +298,74 @@ public AgentSettings LoadSettings()
             }
         }
 
+        public async Task UnconfigureAsync(CommandSettings command)
+        {
+            if (!_store.IsConfigured())
+            {
+                _term.WriteError(StringUtil.Loc("MissingSettings"));
```

Comment:
If the user runs unconfig without "sudo", then the service will not be removed, but the setting file will be removed. Then the user may run unconfig a second time with "sudo". If we return here, we won't try to unregister the service again.
The main reason I print error message here is, because the "delete agent from server" will fail latter and I would like to make it clear to the user, that it is because the setting file is missing. One improvement is to set a boolean flag if setting file is missing and not try to delete the agent from the server, because we know it is going to fail and print another error.